### PR TITLE
Not explicit creation of main config files in settingUp devdoc. #5018

### LINKED ISSUE #5018: Not explicit creation of main config files in settingUp devdoc.
In [settingUp devdoc here](https://github.com/TEAMMATES/teammates/blob/master/devdocs/settingUp.md#setting-up-the-dev-server), the third point (creation of main config files), does not mention explicitly enough the process of creating files without ".template" in the names.

### PR DESCRIPTION
**Pull request by samatdav**
Fixes #5018

### CODE REVIEW

File: devdocs/settingUp.md [hunk 1]
```diff
@@ -43,10 +43,9 @@ Important: When a version is specified, please install that version instead of t
     * HTML syntax: We prefer not to use the HTML syntax validator provided by Eclipse.
     To turn it off, go to `Window → Preferences → Validation → HTML Syntax Validator` and uncheck the `Build` option.
 3. Create main config files {These are not under revision control because their 
-   content vary from developer to developer}.
+   content vary from developer to developer}. In the folders below, duplicate template files and remove ".template" from the names of the duplicates.
```
Comment (thyageshm) [inline devdocs/settingUp.md]: Technically we are directly providing the files to be created, and not the folders.

**Comment by thyageshm** [comment]
In my opinion, the current explanation is pretty clear. One way we could "improve" (based on your feedback) is to change `copy -> paste -> rename` to something like `duplicate -> remove '.template' from name`. What do you think @samatdav @damithc ?

**Comment by damithc** [comment]
Yes, that's the sort of "improvement" I was expecting too.

**Comment by samatdav** [comment]
@thyageshm changed as you proposed

**Comment by thyageshm** [comment]
nice :-)

**Pull request merged**